### ISSUE TITLE
Private key and passwors shows up while deploying encryptKey.js

### ISSUE THREAD

**Issue by defih** [PatrickAlphaC/ethers-simple-storage-fcc#51]
7e4d139eed481a441af9b92301e8399fc08941ff1dff64ec06ed2b1ca74e7dee
password
{"address":"15085b7e9d18f77bbd6cff655659869ec727a479","id":"41ea1350-c30c-4d64-9183-b7c35a072f0a","version":3,"crypto":{"cipher":"aes-128-ctr","cipherparams":{"iv":"7d04a47e5e432eed4be40998aec54d95"},"ciphertext":"3a700c1fe3af4734b3ed1eb8955e3123f46bd3935533568c33e9b62e64093185","kdf":"scrypt","kdfparams":{"salt":"ae4b643339414f8f1687c4de30a8e957d7b52182c7b10547e09239b0864ffc55","n":131072,"dklen":32,"p":1,"r":8},"mac":"3cbba12c07c9770540e3c350dd12bccd8511bbf05cbe9a3751279870099456d4"}}
![croppederror](https://user-images.githubusercontent.com/111440230/202847634-f7963fc7-a3be-426f-a2f3-01db06748bae.PNG)

**Comment by PatrickAlphaC** [comment]
Good point! Can you make a PR to remove the `console.log` lines that print them out?
